tutorials/k8s-kapsule-multi-az/index.mdx

```diff
@@ -0,0 +1,614 @@
+        type     = "A"
+        data     = kubernetes_service.nginx["fr-par-1"].status[0].load_balancer[0].ingress[0].ip
+        ttl      = 60
+
+        http_service {
+            ips = [
+                kubernetes_service.nginx["fr-par-1"].status[0].load_balancer[0].ingress[0].ip,
+                kubernetes_service.nginx["fr-par-2"].status[0].load_balancer[0].ingress[0].ip,
+            ]
+            must_contain = "up"
+            url          = "http://ingress.scw.yourdomain.tld/up"
+            user_agent   = "scw_dns_healthcheck"
+            strategy     = "all"
+        }
+    }
+    ```
+
+6. Apply your configuration and wait for the deployment to be ready.
+    ```
+    terraform apply
+    ```
+
+7. Check now that your nginx deployment is running and that the pods are distributed across the different node pools in the different AZs.
+    ```
+    kubectl get pods -n ingress-nginx -o wide
+
+    NAME                                        READY   STATUS    RESTARTS   AGE   IP             NODE                                             NOMINATED NODE   READINESS GATES
+    ingress-nginx-controller-6878d8d4b6-hkq44   1/1     Running   0          30m   100.64.9.233   scw-kapsule-multi-az-pool-fr-par-1-61e22198f8c   <none>           <none>
+    ingress-nginx-controller-6878d8d4b6-mzc5q   1/1     Running   0          30m   100.64.11.78   scw-kapsule-multi-az-pool-fr-par-1-8334e772ced   <none>           <none>
+    ingress-nginx-controller-6878d8d4b6-cgvt2   1/1     Running   0          30m   100.64.10.76   scw-kapsule-multi-az-pool-fr-par-2-1bcf90f3683   <none>           <none>
+    ingress-nginx-controller-6878d8d4b6-xmqm9   1/1     Running   0          30m   100.64.8.164   scw-kapsule-multi-az-pool-fr-par-2-33265e85597   <none>           <none>
+    ingress-nginx-controller-6878d8d4b6-4wbtj   1/1     Running   0          30m   100.64.7.201   scw-kapsule-multi-az-pool-fr-par-3-44b14b7bbbd   <none>           <none>
+    ingress-nginx-controller-6878d8d4b6-mg9vp   1/1     Running   0          30m   100.64.6.210   scw-kapsule-multi-az-pool-fr-par-3-863491657c7   <none>           <none>
+    ```
+
+8. You can also check that both services of type LoadBalancer are up and get their external IPs.
+    ```
+    kubectl get services -n ingress-nginx 
+
+    NAME                                 TYPE           CLUSTER-IP     EXTERNAL-IP     PORT(S)                      AGE
+    ingress-nginx-controller-admission   ClusterIP      10.42.12.8     <none>          443/TCP                      31m
+    ingress-nginx-controller-fr-par-1    LoadBalancer   10.41.26.168   195.154.68.56   80:32429/TCP,443:30842/TCP   14m
+    ingress-nginx-controller-fr-par-2    LoadBalancer   10.42.96.9     51.159.85.96    80:30490/TCP,443:31351/TCP   14m
+    ```
+
+9. Using your browser or curl command you can check that `ingress-nginx` is available. Since there is no ingress configured for the moment, it should respond with a 404 status code.
```

Comment:
```suggestion
9. Using your browser or curl command, check that `ingress-nginx` is available. Since there is no ingress configured for the moment, it should respond with a 404 status code.
```